alb/alb.go

```diff
@@ -124,6 +124,13 @@ func (a *alb) Health() error {
 	return fmt.Errorf("have not attached to all frontends %v yet", a.targetGroupNames)
 }
 
+func (a *alb) Readiness() error {
+	if !a.readyForHealthCheck.Get() {
+		return errors.New("ALB registration not attempted yet")
+	}
+	return a.Health()
+}
+
 func (a *alb) String() string {
 	return "ELB frontend"
```

Comment:
```suggestion
	return "ALB frontend"
```

Not related to your changes obviously, but while we're in this file we might as well correct this typo.